resources/site/how-to/workbench-application-platform/how-to-issue-custom-intent.md

```diff
@@ -0,0 +1,47 @@
+![SCION Workbench](/resources/site/logo/scion-workbench-banner.png)
+
+[Overview][menu-overview] | [Workbench][menu-workbench] | [Workbench&nbsp;Application&nbsp;Platform][menu-workbench-application-platform] | [Contributing][menu-contributing] | [Changelog][menu-changelog] | [Sponsoring][menu-sponsoring] | [Links][menu-links]
+|---|---|---|---|---|---|---|
+
+## Issuing a custom intent to the host application
```

Comment:
I would remove 'to the host application' because the intent can be handled by any application.

Please use the 'How to ...' form similar to the other how-to's.